VideoLocker/res/values/strings.xml

```diff
@@ -295,6 +295,12 @@
     <string name="discussion_priviledged_author_label_staff">Staff</string>
     <!-- The label to use when indicating a post was authored by a privileged user -->
     <string name="discussion_priviledged_author_attribution">By %s</string>
+    <!-- The label to use when describing the date and author of a discussion thread -->
+    <string name="discussion_thread_attribution">Discussion posted %1$s by %2$s %3$s</string>
+    <!-- The label to use when describing the date and author of a discussion question -->
+    <string name="question_thread_attribution">Question posted %1$s by %2$s %3$s</string>
+    <!-- The label to use when describing the date and author of a discussion comment -->
+    <string name="comment_attribution">Posted %1$s by %2$s %3$s</string>
```

Comment:
Oh weird. I was on a super old version. I think it got lost in a merge? https://github.com/edx/edx-app-android/commit/b2d07359b9773e06efa64d007e197ddb9967f328#diff-843060c125e94dd1417ff2bf53a9189eL43